Travel_fellowships.md

```diff
@@ -1,41 +1,183 @@
+
+OBF promotes the development of open-source bioinformatics software, documentation, or other content useful for research communities. 
+In this section, applicants are asked to provide a link to an online repository and evidence of open-source licensing. 
+
+They are further expected to describe the project that their participation in the event involves (either because they are presenting, co-developing, or because the event involves activities around this topic).
+If they cannot provide links to specific resources and licenses, then they can describe how their participation furthers open science practices.
+
+### Promoting diversity and representation
+
+One of the goals of the OBF fellowship is to increase participation from groups otherwise underrepresented in the community or at the event, including but not limited to underrepresented demographic groups (country of residence and citizenship), ethnic background (historically underrepresented and other minority groups), career stages, gender identity and expression, and people with disabilities.
+
+In an open text response, applicants can describe how their participation in this event help increase diversity and representation of any marginalised and traditionally underrepresented group in the OVERALL open source bioinformatics and/or open science community.
+
+Note that applicants may have an answer to this question even if they are not personally a member of an underrepresented group. 
+This section is important for the evaluation of their application.
+
+### Justification for funding
+
+The OBF Event Fellowships are available for the awardees to cover conference registration fees and other participation related costs of up to USD $1,000. 
+  - For the in-person events, this fellowship can be used to cover airfare, transportation and hotel. 
+  - For the virtual events, applicants can seek support for small hardware, internet cost for the duration of the event, or other supporting materials
+
+Applicants are asked to provide details of the items they would like to pay for through the OBF Event fellowship. 
+Here, they can provide as much detail as possible to help the review panel make a fair recommendation for the fellowship awardees.
+
+Please see the next section for more details.
+
+## What does the event fellowship cover?
+
+The OBF Event Fellowships are available for the awardees to cover conference registration fees and other participation related costs of up to USD $1,000. 
+
+### Items covered for in-person participation
+
+The fellowship can be used to cover conference registration fees and direct travel costs (such as airfare, train, metro, bus and taxi), hotel, and/or child-care up to a value of USD 1,000. 
+Note that rental car expenses and personal car mileage are not eligible for reimbursement. 
+If you anticipate that you would require more than USD 1,000 in order to attend the event, please include that information on the application form and contact the fellowship chair.
```

Comment:
i think this is the point where the document suddenly switches from the third person ("the applicant") to the second ("you"). i suggest choosing one of those and using it consistently throughout.